content/api/templates.apib

```diff
@@ -529,22 +422,19 @@ See the [Substitutions Reference section](/api/substitutions-reference/) for mor
                 "text": "Check out these deals Natalie!",
                 "html": "<b>Check out these deals Natalie!</b>",
                 "headers": {
-                    "X-Customer-Campaign-ID": "Summer2014"
+                    "X-Example-Header": "Summer2018"
                 }
             }
         }
 
-## Delete [/templates/{id}]
-
-### Delete a Template [DELETE]
+### Delete a Template [DELETE /templates/{id}]
 
-Delete a template by specifying its ID in the URI path.
-If the template delete API call succeeds, then ALL versions of the template will be deleted from the system (both published AND draft versions).
+Deletes all versions, both published and draft versions, of a template.
```

Comment:
"Deletes both published and draft versions of a template."? To simplify the sentence.